perl/ANNOTATION.md

```diff
@@ -0,0 +1,781 @@
+
+`dbSNP.MGP`: MGP contains aggregated information on 267 healthy individuals, representative of the Spanish population that were used as controls in the MGP (Medical Genome Project).
+
+<br/>
+
+#### [cadd](http://cadd.gs.washington.edu)
+
+A score >=0 that indicates deleteriousness of a variant. Variants with cadd > 15 are more likely to be deleterious.
+See http://cadd.gs.washington.edu.
+
+<br/>
+
+#### [caddIndel](http://cadd.gs.washington.edu)
+
+A score >=0 that indicates deleteriousness of a variant. Variants with cadd > 15 are more likely to be deleterious.
+See http://cadd.gs.washington.edu.
+
+caddIndel scores are only defined for indels and MNPs. For SNPs, use the `cadd` field.
+
+- Note that because Bystro decomposes MNPs into "SNP" records, the `caddIndel` field will occasionally be populated for SNPs (which are in fact part of MNPs in the CADD Indel dataset).
+
+`caddIndel.alt`: The Bystro VCF-preprocessor's ALT record for this CADD site. This should always match the row's `alt` field value
+
+`caddIndel.id`: The CADD VCF `ID`
+
+`caddIndel.PHRED`: The CADD PHRED score for the insertion or deletion
+
+<br/>
+
+#### clinvarVcf
+
+ClinVar annotations, sourced from the ClinVar VCF dataset
+
+`clinvarVcf.id`: The ClinVar VCF `ID`
+
+`clinvarVcf.alt`: The Bystro VCF-preprocessor's ALT record for this ClinVar site. This should always match the row's `alt` field value
+
+`clinvarVcf.AF_ESP`: Allele frequencies from GO-ESP
+
+`clinvarVcf.AF_EXAC`: Allele frequencies from ExAC
+
+`clinvarVcf.AF_TGP`: Allele frequencies from TGP
+
+`clinvarVcf.ALLELEID`: The ClinVar Allele ID
+
+`clinvarVcf.CLNDN`: ClinVar's preferred disease name for the concept specified by disease identifiers in CLNDISDB
+
+`clinvarVcf.CLNDNINCL`: For included Variant : ClinVar's preferred disease name for the concept specified by disease identifiers in CLNDISDB
+
+`clinvarVcf.CLNHGVS`: Top-level (primary assembly, alt, or patch) HGVS expression
+
+`clinvarVcf.CLNREVSTAT`: ClinVar review status of germline classification for the Variation ID
+
+`clinvarVcf.CLNSIG`: Aggregate germline classification for this single variant;
+
+`clinvarVcf.CLNSIGCONF`: Conflicting germline classification for this single variant
+
+`clinvarVcf.CLNVCSO`: Sequence Ontology id for variant type
+
+`clinvarVcf.DBVARID`: NSV accessions from dbVar for the variant
+
+`clinvarVcf.ORIGIN`: Allele origin. One or more of the following values may be added: 0 - unknown; 1 - germline; 2 - somatic; 4 - inherited; 8 - paternal; 16 - maternal; 32 - de-novo; 64 - biparental; 128 - uniparental; 256 - not-tested; 512 - tested-inconclusive; 1073741824 - other
+
+`clinvarVcf.RS`: dbSNP ID (i.e. rs number)
+
+// add logofunc.id logofunc.alt logofunc.prediction logofunc.neutral logofunc.gof logofunc.lof genebass.id genebass.alt genebass.phenocode genebass.description
+
+<br/>
+
+#### (hg38-only) [LoGoFunc](
+
+https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10688473/)
```

Comment:
Just checking, should this be on the same line as the title for this section or is there a reason why it's below?